app/components/primer/beta/auto_complete/auto_complete.html.erb

```diff
@@ -1,6 +1,13 @@
 <%= label %>
 <%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <% if icon.present? %>
+    <label for="<%= @input_id %>">
```

Comment:
what happens if we have both `label` and `icon`? will it have two `<label>`?

Reply:
As it is currently written, yes it will have two labels. We have an incoming change to the `auto-complete-element` that will change this design.